src/NHibernate.Test/TypesTest/TimestampTypeFixture.cs

```diff
@@ -24,7 +24,7 @@ public void ObsoleteMessage()
 					var log = spy.GetWholeLog();
 					Assert.That(
 						log,
-						Does.Contain($"NHibernate.Type.TimestampType ({NHibernateUtil.Timestamp.Name}) is obsolete. Please use DateTimeType instead.").IgnoreCase);
+						Does.Contain($"NHibernate.Type.TimestampType is obsolete. Please use DateTimeType instead.").IgnoreCase);
```

Comment:
Source and message of log is changed. Before:
https://github.com/nhibernate/nhibernate-core/blob/f027dad690566f7fbf7d4938a897b708f57f3dfd/src/NHibernate/Type/TypeFactory.cs#L690
Now:
https://github.com/nhibernate/nhibernate-core/blob/f027dad690566f7fbf7d4938a897b708f57f3dfd/src/NHibernate/Type/TypeFactory.cs#L549-L550

I don't think we should really care about keeping old message - so I just adjusted test